keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -498,7 +509,12 @@ the existing API objects?**
   Yes.
   
   - API type(s): Job
-  - Estimated increase in size: new field of about 30 bytes.
+  - Estimated increase in size:
+    - New field in Spec about 30 bytes.
+    - New field in Status. In the worst case scenario, completed indexes are
+      non-consecutive. Since parallelism is limitted to 10^5, we could have
+      up to 5*10^4 non-consecutive numbers, which would lead to a size of
+      up to 2.5Mb
```

Comment:
`1-2,4-5,7-8,10-11,13-14,...` (3.14 Mbits) takes more space than `1,3,5,7,9,...` (2.36 Mbits)

Reply:
https://play.golang.org/p/rkMIh1nOq1B

Reply:
Why 2.5MB? Also - 2.5MB is more than our current hard-limit, so this isn't fine.

Reply:
I think that's 2.5 megabits (~300 kilobytes), not 2.5 megabytes.

Reply:
I'm not looking to be precise here. But it looks like size is much smaller from @adtac simulations. We don't need to count bits, just bytes.

I expanded the simulation for ~1B completions (https://play.golang.org/p/cZiqILUlpqv). And worst case doesn't get to 1Mb.

Reply:
That's 1 MB (uppercase B), not 1 Mb (lowercase b), which is the source of confusion here I think.

Reply:
fixed